src/main.rs

```diff
@@ -14,22 +14,41 @@ mod epic_info;
 #[tokio::main]
 async fn main() -> Result<(), Box<dyn std::error::Error>> {
     dotenv::dotenv().ok();
-    let matches = App::new("main")
+    let matches = Command::new("main")
         .about("Creates a visual diagram of a Pivotal Tracker epic")
         .arg(
-            Arg::with_name("epic_name")
-                .short("e")
+            Arg::new("epic_name")
+                .short('e')
```

Comment:
New syntax for the upgraded `clap` version.
And short is restricted to actually take a character now.